runtime/oti/stackwalk.h

```diff
@@ -142,7 +142,14 @@ extern "C" {
 #define J9_STACK_FLAGS_JIT_FAILED_METHOD_MONITOR_ENTER_RESOLVE 0x01000000
 #define J9_STACK_FLAGS_JIT_EXCEPTION_CATCH_RESOLVE 0x01100000
 
-/* Platform-specific defines */
+/* Platform-specific defines
+ *
+ * - J9SW_JIT_STACK_SLOTS_USED_BY_CALL
+ *     Number of slots pushed on a stack by call itself. Should be 0 (zero)
+ *     for architectures that store return address into a link register.
+ *     Currently only X86 defines it as 1 since x86 `call` instruction pushed
```

Comment:
grammar: pushed -> pushes

Is the value for RISC-V incorrect? If so it should be corrected. If not this sentence should say "... only RISC-V and X86 define ...".

Reply:
> grammar: pushed -> pushes

Thanks, fixed in 1063962

> Is the value for RISC-V incorrect?

Yes, the value was incorrect. I have another commit fixing this which I intended to submit once this gets in. 
I have added it to this PR as requested: c819f06